course/admin.py

```diff
@@ -62,11 +63,26 @@ class EnrollmentAdmin(admin.ModelAdmin):
     list_filter = ("course_instance",)
 
 
+# Filtering obejct to help scheduling maintenance breaks
+class FutureDeadlines(admin.SimpleListFilter):
+    title = _('Future deadlines')
+    parameter_name = 'future_deadlines'
+
+    def lookups(self, request, model_admin):
+        return (
+            ('now', _("Show only modules with 'closing_time in the' future.")),
+        )
+
+    def queryset(self, request, queryset):
+        if self.value() == 'now':
+            return queryset.filter(closing_time__gte=timezone.now()).order_by('closing_time')
```

Comment:
The `late_submission_deadline` should also be included here in the query set filter since students may still submit at that time. It must also check that the late submissions are enabled since otherwise, the late deadline would be meaningless (it could have a value but the date is never used if the late submissions are disabled).